packages/shared/src/components/modals/post/CommentModal.tsx

```diff
@@ -65,6 +48,37 @@ const getCommentFromCache = ({
   return undefined;
 };
 
+interface GetCommentFromCacheProps {
+  client: QueryClient;
+  postId: string;
+  commentId?: string;
+}
+
+const getCommentFromCache = ({
+  client,
+  postId,
+  commentId,
+}: GetCommentFromCacheProps): Comment | undefined => {
+  if (!commentId) {
+    return undefined;
+  }
+
+  const keys = getAllCommentsQuery(postId);
+  // eslint-disable-next-line no-restricted-syntax
+  for (const key of keys) {
```

Comment:
Do you need a for loop here even? you only care about one right? 
Maybe we can just use `keys[0]`? 

Or is there some high likelyhood of it not existing?

Reply:
Yes, we only need now, but we return early so we won't have to worry to iterating everything.